Data/Chimera.hs

```diff
@@ -239,13 +265,41 @@ tabulateFixM f = result
         fixF k
           | k < int2word ii
           = flip index k <$> result
-          | k < int2word ii `shiftL` 1
+          | k <= int2word ii `shiftL` 1 - 1
```

Comment:
Nice catch! Is it possible to write a specific test, which triggers the bug?

Reply:
I think this one is tricky, because we would need to index into the very last vector. Maybe if we factor out the bounds calculation into another function and test that one

Reply:
Ah, I misread the change. We could potentially write a test for 32-bit architecture, but probably not worth it. Nevermind.